c/httpserver.c

```diff
@@ -6001,7 +6025,27 @@ int mainHttpLoop(HttpServer *server){
   return stcBaseMainLoop(base, MAIN_WAIT_MILLIS);
 }
 
-
+void registerHttpAuthorizationHandler(HttpServer *server, int authorizationType, AuthorizationHandler *authorizationHandler, void *userData) {
```

Comment:
Would we want to return an int to indicate success/failure? Maybe 0 is a success, and -1 (or some other non-zero) is a failure (in your case an allocation failure). There are functions in `httpserver.c` that seem to do that.

Reply:
I've added a return code for `registerHttpAuthorizationHandler`.